src/components/MainSlide/index.tsx

```diff
@@ -0,0 +1,35 @@
+import Horizontal from "@/components/Card/Horizontal";
+import Socials from "@/components/Card/Horizontal/Socials";
+import Caption from "@/components/Caption";
+import Button from "@/components/Button";
+
+import useDetactMobile from "@/hooks/useDetactMobile";
+
+import styles from "./MainSlide.module.scss";
+
+interface Props {
+  caption?: JSX.Element;
+  title: JSX.Element;
+  description?: JSX.Element;
+  image?: JSX.Element;
+  btnMsg?: JSX.Element;
+}
+function MainSlide({ title, description, image, btnMsg }: Props) {
+  const isMobile = useDetactMobile();
```

Comment:
can this be detect instead of detact pls?

Reply:
Oh no.. again.. I should stop making typos!